core/utils/utils_example_test.go

```diff
@@ -1,48 +0,0 @@
-package utils
```

Comment:
all of the tests have been moved to common repo?

Reply:
The one `ExampleStopChan` in this repo isn't really doing anything interesting, and I think that I never really completed it. It was intended to demonstrate the API, but that seems unecessary :shrug:.

Reply:
oh `utils_test.go` had regular tests, and those were already copied with the aliased types.